src/library.js

```diff
@@ -3494,6 +3494,22 @@ LibraryManager.library = {
 #endif
   },
 
+  emscripten_out__sig: 'vi',
+  emscripten_out: function(str) {
```

Comment:
Do these need the `_` prefix to match the C++ names?
```suggestion
  _emscripten_out__sig: 'vi',
  _emscripten_out: function(str) {
```